shepherd.yaml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+  shepherd:
+    image: atsigncompany/shepherd:latest
+    environment:
+      TZ: 'US/PST'
```

Comment:
I get that this is your local TZ, but server stuff should normally be UTC

Reply:
agreed